docs/package_index_json-specification.md

```diff
@@ -80,6 +80,7 @@ Each tool describes a binary distribution of a command line tool. A tool can be:
 - a file preprocessor
 - a debugger
 - a program that performs a firmware upgrade
+- a [Pluggable Discovery](pluggable-discovery-specification.md)
```

Comment:
I wasn't sure either but decided to go with the capitalized one since it's the name of the feature. But am ok if we want to make it lowercase.

Reply:
I don't have a super strong opinion on which way it should be, but I think this is the time to make a conscious decision and follow it consistently from here on. Otherwise, conscientious writers may waste time trying to determine the correct way to write this term in the face of inconsistent precedents.

@cmaglie do you have a preference re: "Pluggable Discovery" vs. "pluggable discovery"?

Reply:
Oops, I forgot that Cristian is away right now.

Reply:
Yep, it's up to us. Probably it's going to be better to use the lowercase one, as you said it's not a proper noun so it doesn't come natural capitalizing it. :shrug: